### DISCUSSION TITLE
SQLITE_ERROR: foreign key mismatch

### DISCUSSION THREAD

**Discussion by Zulqis** [discord-tickets/bot#228] [Support (Q&A)] [answered]
![image](https://user-images.githubusercontent.com/88617135/131592437-296dbdc3-6f8d-4ac2-a991-711e9370862b.png)
 
 hello, i'm newbie and dumb with this.
I get this problem after close a ticket, maybe someone can help me with this? thanks!

**Comment by eartharoid** [comment, marked as answer]
This is a known issue (#162) and you can ignore it as it shouldn't affect functionality.
